### PR TITLE
Add Spectrum IT as platinum sponsor

### PR DESCRIPTION
**Pull request by lboynton**

### CODE REVIEW

**Comment by asgrim** [comment]
Just waiting for confirmation on the text to be used.

File: sponsors.html [hunk 1]
```diff
@@ -20,11 +20,52 @@
...
+        <p>
+            <a href="https://www.spectrumit.co.uk" target="_blank" rel="noopener noreferrer">
+                <img src="/img/sponsors/spectrumit.png" class="img-responsive center-block" title="Spectrum IT"
+                     alt="Spectrum IT"/>
+            </a>
+        </p>
+        <div class="row text-center">
+            <div class="col-md-8 col-md-offset-2">
+                <p class="lead">
+                    Spectrum IT Recruitment is an IT &amp; technology focused recruitment consultancy working primarily
```
Comment (asgrim) [inline sponsors.html]: Should read: 
> Spectrum IT Recruitment is a specialist provider of IT and Technology recruitment to companies across the Hampshire & the South. Our expertise spans across web development, software engineering and digital environments and we boast a wealth of job opportunities with the region’s leading employers. Spectrum IT is committed to supporting the local IT community, creating a positive experience for job seekers and providing our candidates with the platform to achieve their career goals and aspirations.

**Review by asgrim — CHANGES_REQUESTED** [review]

File: sponsors.html (hunk 1 above)
Comment (lboynton) [reply sponsors.html]: Got rid of the first "the" in 

> the Hampshire & the South

**Comment by lboynton** [comment]
Forgot to mention, couple of responsiveness improvementness in this as well

**Review by asgrim — APPROVED** [review]

**Comment by asgrim** [comment]
:+1:

**Pull request merged**